### PR TITLE
Trams api trust search

### PR DESCRIPTION
**Pull request by MarkRobinsonDfe**

### CODE REVIEW

**Review by wystanpalm-madetech — CHANGES_REQUESTED** [review]
Please remove the fields that you're not using the search result - I think it's probably only a handful of them

**Review by wystanpalm-madetech — APPROVED** [review]

File: Dfe.Academies.External.Web/AcademiesAPIResponseModels/Schools/EstablishmentResponse.cs [hunk 1]
```diff
@@ -1,5 +1,3 @@
-using System.Text.Json.Serialization;
-
 namespace Dfe.Academies.External.Web.AcademiesAPIResponseModels.Schools;
 
 public record EstablishmentResponse
```
Comment (wystanpalm-madetech) [inline Dfe.Academies.External.Web/AcademiesAPIResponseModels/Schools/EstablishmentResponse.cs:3]: There are nullable related warnings in this class, I think we could resolve this by removing the default constructor and changing the implementation of the ResilientRequestProvider to use:

```
return await _client.GetFromJsonAsync<TResult>(uri);
```

**Review by wystanpalm-madetech — CHANGES_REQUESTED** [review]

File: Dfe.Academies.External.Web/AcademiesAPIResponseModels/Schools/EstablishmentResponse.cs (hunk 1 above)
Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web/AcademiesAPIResponseModels/Schools/EstablishmentResponse.cs:3]: ?? can we catch up about this tomo @wystanpalm-madetech ?

Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web/AcademiesAPIResponseModels/Schools/EstablishmentResponse.cs:3]: Amended ResilientRequestProvider Get<> method @wystanpalm-madetech

**Review by wystanpalm-madetech — APPROVED** [review]

**Pull request merged**